build/yamls/antrea-config.yml

```diff
@@ -0,0 +1,211 @@
+apiVersion: v1
```

Comment:
why is it needed? is it redundant with antrea.yaml if both are provided to e2e tests?

Reply:
we'd like to skip test faster before antrea deployment actually happens, so need a way to get the antrea-agent.conf and antrea-controller.conf. and antrea.yaml concludes lots of other yaml definitions and it's not easy to extra antrea-config.yaml  through yaml unmarshal which doesn't support multiple files.